client/ayon_core/hosts/max/plugins/publish/collect_workfile.py

```diff
@@ -48,15 +39,14 @@ def process(self, context):
         })
 
         data['representations'] = [{
-            'name': ext.lstrip("."),
-            'ext': ext.lstrip("."),
+            'name': ext,
+            'ext': ext,
             'files': file,
             "stagingDir": folder,
         }]
 
         instance.data.update(data)
-
-        self.log.info('Collected instance: {}'.format(file))
-        self.log.info('Scene path: {}'.format(current_file))
-        self.log.info('staging Dir: {}'.format(folder))
-        self.log.info('subset: {}'.format(subset))
+        self.log.debug('Collected data: {}'.format(data))
+        self.log.debug('Collected instance: {}'.format(file))
+        self.log.debug('Scene path: {}'.format(current_file))
+        self.log.debug('staging Dir: {}'.format(folder))
```

Comment:
```suggestion
        self.log.debug('Collected workfile path: {}'.format(current_file))
        self.log.debug('Collected workfile staging dir: {}'.format(folder))
        self.log.debug('Collected workfile data: {}'.format(data))
```
Would this be sufficient debug data to log?

Also, you're using quite a mixture of `'` quotes versus `"` in this file. Should we align them to all use `"` like most of the codebase?